modules/gdscript/language_server/gdscript_text_document.cpp

```diff
@@ -56,6 +57,9 @@ void GDScriptTextDocument::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("show_native_symbol_in_editor"), &GDScriptTextDocument::show_native_symbol_in_editor);
 }
 
+void GDScriptTextDocument::didClose(const Variant &p_param) {
+}
```

Comment:
Could go after `didOpen` to be consistent with the declarations.

Might be worth adding a oneline comment to confirm that this is empty by choice and not by oversight? (i.e. there's nothing to do in Godot for this notification)

Reply:
Sure